frontend/src/renderer/pages/file-upload/components/Urlloader.tsx

```diff
@@ -40,8 +42,15 @@ const form = useForm({
     console.log(payload);
     apiCall('/files/website', 'POST', payload)
       .then((response) => {
-        console.log(response.data);
-        window.location.reload();
+
+        Notifications.show({
+          title: 'URL Upload Successful',
+          message: 'The URL was successfully uploaded.',
+          color: 'green'
+        });
+        
+        // passing the website to the Upload.tsx component
+        onFilesUploaded([payload.website]);
       })
```

Comment:
we should add a check here, if the status code f.e. is 500 it still will "upload successful".